bazel/oci/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+readonly OCI_REPOSITORY=$1
+readonly BAZEL_VERSION=$2
+
+set -o errexit -o nounset -o pipefail
+
+function print_usage() {
+    >&2 echo "Usage: $0 <OCI_REPOSITORY> <BAZEL_VERSION>"
+}
+
+if [ -z "${OCI_REPOSITORY}" ]; then
+    >&2 echo "ERROR: missing 'OCI_REPOSITORY' argument"
+    print_usage
+    exit 1
+fi
+
+if [ -z "${BAZEL_VERSION}" ]; then
+    >&2 echo "ERROR: missing 'BAZEL_VERSION' argument"
+    print_usage
+    exit 1
+fi
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+readonly GIT_ROOT
+
+if docker buildx version 2>&1 1>/dev/null; then
```

Comment:
Almost surely does not work as expected:
- https://stackoverflow.com/questions/18012930

Reply:
Thanks for spotting that. I'll fix it!

Reply:
https://github.com/bazelbuild/continuous-integration/pull/1315

Reply:
For future references, the official documentation is more relevant than StackOverflow: https://www.gnu.org/software/bash/manual/html_node/Redirections.html.